postinstall.js

```diff
@@ -5,25 +5,27 @@
 var sysPath = require('path');
 var fs = require('fs');
 
-var dist = sysPath.join(__dirname, 'dist');
+try {
+  var dist = sysPath.join(__dirname, 'dist');
 
-if (!fs.existsSync(dist)) {
-  fs.mkdirSync(dist);
-}
+  if (!fs.existsSync(dist)) {
+    fs.mkdirSync(dist);
+  }
 
-var runtime = ['', '.runtime'], amd = ['', '.amd'], min = ['', '.min'];
-var hPath = sysPath.dirname(require.resolve('handlebars'));
+  var runtime = ['', '.runtime'], amd = ['', '.amd'], min = ['', '.min'];
+  var hPath = sysPath.dirname(require.resolve('handlebars'));
 
-runtime.forEach(function(r) {
-  amd.forEach(function(a) {
-    min.forEach(function(m) {
-      var fileName = 'handlebars' + r + a + m + '.js';
-      var absFilePath = sysPath.join(hPath, '..', 'dist', fileName);
-      var file = sysPath.join(__dirname, 'dist', fileName);
-      if (!fs.existsSync(file)) {
-        fs.createReadStream(absFilePath).pipe(fs.createWriteStream(file));
-      }
+  runtime.forEach(function(r) {
+    amd.forEach(function(a) {
+      min.forEach(function(m) {
+        var fileName = 'handlebars' + r + a + m + '.js';
+        var absFilePath = sysPath.join(hPath, '..', 'dist', fileName);
+        var file = sysPath.join(__dirname, 'dist', fileName);
+        if (!fs.existsSync(file)) {
+          fs.createReadStream(absFilePath).pipe(fs.createWriteStream(file));
+        }
+      });
     });
   });
-});
+} catch (e) {}
```

Comment:
I'd be ok with that explanation if the `try` were more targeted.

I'm not opposed to this general idea, but it's worth writing `something` to the console since any type of error could have occurred.

Actually, `process.on('uncaughtException', ` might be the more appropriate way to do this.